packages/payload/src/fields/config/types.ts

```diff
@@ -114,6 +114,7 @@ type Admin = {
   description?: Description
   disableBulkEdit?: boolean
   disabled?: boolean
+  disabledListColumn?: boolean
```

Comment:
Let's actually rename this to `disableListColumn` instead (no d on disabled) to be consistent with `disableBulkEdit`.